code/src/index.css

```diff
@@ -5,9 +5,158 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #FFD831;
+  box-sizing: border-box;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+  margin: 1.5rem auto 1.5rem auto;
+  max-width: 520px;
+  min-width: 350px;
+}
+
+.logo {
+  width: 100%;
+}
+
+main {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+  min-width: 340px;
+  max-width: 500px;
+  font-size: 1rem;
+  margin: 1.5rem auto 1rem auto;
+  padding: 10px 12px 5px 10px;
+  background-color: #FFD831;
+}
+
+.form-section {
+  border: black solid 3px;
+  border-radius: 3px;
+  background-color: #A6A6A6;
+  box-shadow: 0.5rem 0.5rem rgb(31, 31, 31);
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+}
+
+.label {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+}
+
+.thought-input {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+  margin: 1rem 0 0 0;
+  padding: 0.5rem;
+  font-size: 1rem;
+}
+
+.character-count,
+.character-warning {
+  margin: 0.3rem 0 0 0;
+  text-align: center;
+  font-style: italic;
+  font-size: 1rem;
+  color:#6b6b6b;
+  margin-bottom: 1rem;
+}
+
+.submit-button {
+  background-color: #FFD831;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+  padding: 1em 1rem;
+  font-weight: 700;
+  cursor: crosshair;
+  color: black;
+  border-color: black;
+}
+
+.submit-button:hover {
+  background-color: black;
+  color: #FFD831;
+}
+
+.thought-section {
+  margin-top: 4.5rem;
+}
+
+.single-thought {
+  border: black solid 3px;
+  border-radius: 4px;
+  background-color: white;
+  margin: 1.5rem 1rem 1rem 1rem;
+  box-shadow: 0.5rem 0.5rem rgb(31, 31, 31);
+  padding: 2rem;
+}
```

Comment:
I suggest you try adding `overflow-wrap: break-word; ` to your .single-though to break the words in the container. Now they are over flowing.